forks/Microsoft.Health.FellowOakDicom/Core/DicomElement.cs

```diff
@@ -0,0 +1,185 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using FellowOakDicom;
+using FellowOakDicom.IO.Buffer;
+
+namespace Microsoft.Health.FellowOakDicom.Core;
+
+
+/// <summary>Signed Very Long (SV)</summary>
+public class DicomSignedVeryLong : DicomMultiStringElement
+{
+    #region FIELDS
+
+    private long[] _values;
+
+    #endregion
+
+    #region Public Constructors
+
+    public DicomSignedVeryLong(DicomTag tag, params long[] values)
+        : base(tag, values.Select(x => x.ToString()).ToArray())
+    {
+    }
+
+    public DicomSignedVeryLong(DicomTag tag, IByteBuffer data)
+        : base(tag, null, data)
+    {
+    }
+
+    #endregion
+
+    #region Public Properties
+
+    public override DicomVR ValueRepresentation => DicomVR.SV;
+
+    #endregion
+
+    #region Public Members
+
+    public override T Get<T>(int item = -1)
```

Comment:
I am not sure why we are overriding the whole methos here, why can't we just use the second constructor which takes byeBuffer

Reply:
We are using ByteBuffer constructor. But when we call "GetString" it would call the Get (virtual) method, and the base class implementation does not handle a case, which requires a override in the derived class. (I modeled this similar to how DS and IS VR types are being handled already)

Reply:
I looked into this as well, and there really isn't a way around overriding the `Get` method because it's used on writing. Even if we pass the IByteBuffer in successfully, the json writer will call `Get` on the element, which will not handle the value as a string.